products/db-checkbook/test/test_build_output.py

```diff
@@ -108,7 +133,51 @@ def test_high_sensitivity_fixed_asset():
             "has_geometry",
         ]
     ]
-
-    result = _assign_final_category(clean_join).set_index("fms_id").sort_index()
-    expected_result = generate_expected_final_data().set_index("fms_id").sort_index()
-    assert result.equals(expected_result)
+    historical_liquidations = (
+        _assign_final_category(clean_join).set_index("fms_id").sort_index()
+    )  # align expected and actual results rowwise
+    expected_historical_liquidations = (
+        generate_expected_final_data().set_index("fms_id").sort_index()
+    )  # align expected and actual results rowwise
+
+    def test_columns(self):
+        assert self.historical_liquidations.columns.equals(
+            self.expected_historical_liquidations.columns
+        )
+
+    def test_rows(self):
+        assert (
+            self.historical_liquidations.shape[0]
+            == self.expected_historical_liquidations.shape[0]
+        )
+
+    def test_cp_category(self):
+        assert self.historical_liquidations["cp_category"].equals(
+            self.expected_historical_liquidations["cp_category"]
+        )
+
+    def test_bc_category(self):
+        assert self.historical_liquidations["bc_category"].equals(
+            self.expected_historical_liquidations["bc_category"]
+        )
+
+    def test_cpdb_category(self):
+        assert self.historical_liquidations["cpdb_category"].equals(
+            self.expected_historical_liquidations["cpdb_category"]
+        )
+
+    def test_final_category(self):
+        assert self.historical_liquidations["final_category"].equals(
+            self.expected_historical_liquidations["final_category"]
+        )
+
+    def test_all_categories(self):
+        cols = ["cpdb_category", "bc_category", "cp_category", "final_category"]
+        assert self.historical_liquidations[cols].equals(
+            self.expected_historical_liquidations[cols]
+        )
```

Comment:
does this one cover the 4 column-specific tests above?

Reply:
gotcha. using `assert_frame_equal` in the all category test _might_ give results informative enough to isolate a problem with a specific category, but feels like a nice-to-have thing to maybe try later